modules/blogging/src/Volo.Blogging.Application.Contracts.Shared/Volo/Blogging/BloggingFileContainer.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Volo.Abp.BlobStoring;
+
+namespace Volo.Blogging
+{
+    [BlobContainerName("blogging-files")]
+    public class BloggingFileContainer
```

Comment:
Storing BLOBs/images is an internal detail of the application layer and should not be shared with the clients. So, we should remove any BLOB related stuff from the Application.Contracts and move it to the Application project.